packages/gatsby-admin/README.md

```diff
@@ -0,0 +1,18 @@
+# Gatsby Admin
+
+A visual interface to configure your Gatsby site.
+
+We have not packaged this nicely yet, so it is not installable.
```

Comment:
Your post on Slack indicated that it's now possible. So is this info outdated or that feature in another PR/branch?

Reply:
In another PR (#23368), but @freiksenet pointed out that we don't actually want a plugin as we have to hook into the parent process—so I have to finish #22600 (which requires finishing #22759, which is what I'm working on right now) first 😬